.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+SHARED:
+  &installExosphereStep
+  run:
+    name: install exosphere
+    command: |
+      curl --output exo --location --fail https://github.com/Originate/exosphere/releases/download/v0.35.0/exo-linux-amd64
+      chmod +x exo
+      sudo mv exo /usr/local/bin/
+
+version: 2
+jobs:
+  test:
+    working_directory: ~/edx
```

Comment:
`~/exosphere`?

Reply:
I'll go with `~/space-tweet`.  It doesn't matter that much